src/sssom/parsers.py

```diff
@@ -556,7 +561,7 @@ def from_sssom_rdf(
 
     ms.mappings = mlist  # type: ignore
     _set_metadata_in_mapping_set(mapping_set=ms, metadata=meta)
-    mdoc = MappingSetDocument(mapping_set=ms, prefix_map=prefix_map)
+    mdoc = MappingSetDocument(mapping_set=ms, prefix_map=converter.prefix_map)
```

Comment:
Now I get it. Great.

Reply:
Next step will be replace the prefix_map object in the mapping set document with a converter directly